src/main/xerus/monstercat/tabs/TabSound.kt

```diff
@@ -0,0 +1,65 @@
+package xerus.monstercat.tabs
+
+import javafx.application.Platform
+import javafx.scene.layout.HBox
+import javafx.scene.layout.VBox
+import javafx.scene.control.*
+import javafx.scene.media.AudioEqualizer
+import javafx.scene.media.EqualizerBand
+import javafx.geometry.Orientation
+import xerus.ktutil.javafx.*
+import xerus.ktutil.javafx.properties.*
+import xerus.monstercat.Settings
+import xerus.monstercat.api.Player
+
+class TabSound : VTab() {
+	var hint: Label? = null
+	var eqBox: HBox? = null
+	var eqModel: MutableList<Double> = mutableListOf()
+
+	init {
+		add(CheckBox("Enable Equalizer").bind(Settings.ENABLEEQUALIZER))
+		Player.playerListeners.add { Platform.runLater(::updateEQBox) }
+
+		hint = Label("Play a song to display the controls")
+		add(hint!!)
```

Comment:
`hint = Label("Play a song to display the controls").run { add(it) }`
So vermeidest du das `!!`

Reply:
Guter Hinweis, habe mich mit deinen Extensionmethoden noch nicht so sehr auseinandergesetzt (hauptsächlich aus Zeitgründen)

Reply:
solltest du machen - der Code wird so viel schöner und einfacher dadurch - Extensions sind ein Hauptgrund meiner Liebe zu Kotlin